identity-issuance/src/main/java/com/android/identity/issuance/funke/RequestCredentialsUsingKeyAttestation.kt

```diff
@@ -38,12 +38,12 @@ class RequestCredentialsUsingKeyAttestation(
     fun sendCredentials(
         env: FlowEnvironment,
         credentialRequests: List<CredentialRequest>,
-        keysAssertion: DeviceAssertion // holds AssertionBingingKeys
+        keysAssertion: DeviceAssertion? // holds AssertionBingingKeys
     ): List<KeyPossessionChallenge> {
         credentialRequestSets.add(CredentialRequestSet(
             format = format!!,
             keyAttestations = credentialRequests.map { it.secureAreaBoundKeyAttestation },
-            keysAssertion = keysAssertion
+            keysAssertion = keysAssertion!!
```

Comment:
It appears the null value is treated as a special forking signal now. But here (and in other similar method that signal would throw RT NPE. Is that as expected?

Reply:
Forking signal is actually in `CredentialConfiguration.keyAssertionRequired`. If that is true, keysAssertion **must** be provided. If it is null, this will throw and at some point will be converted to HTTP status 500 - not the best error code, of course, but we did not do a pass through our APIs to make sure all error codes and messages are what we want them to be.